packages/shared/providers.yaml

```diff
@@ -5339,3 +5339,27 @@ falai:
             type: string
             title: API Key
             description: The API key for your fal.ai account
+
+acuity-scheduling:
+    display_name: Acuity Scheduling
+    categories:
+        - scheduling
+        - appointments
+    auth_mode: OAUTH2
+    authorization_url: https://acuityscheduling.com/oauth2/authorize
+    token_url: https://acuityscheduling.com/oauth2/token
+    connection_configuration:
+        - userId
+        - apiKey
+    post_connection_script: acuityPostConnection
+    webhook_routing_script: acuityWebhookRouting
+    proxy:
+        base_url: https://acuityscheduling.com/api/v1
+        decompress: true
+        paginate:
```

Comment:
I was unable to find detailed information on pagination in the provider's documentation. Would you be able to point me to the relevant resource? The docs only mention `max` as a query parameter for the `/appointments` endpoint, but there is no example response so that we would know what to pass as pagination fields for Nango. Any guidance would be greatly appreciated.